scripts/covid19_india/medical_tests_in_data/preprocess.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import csv
+import urllib.request
+
+# 1. All counts are at India level, so geographic reference code will always be IN
+# 2. Only cumulative counts are available at this point
+
+INDIA = "IN"
```

Comment:
Just FYI, if we used `"dcid:country/IND"` instead of `"IN"`, then we wouldn't have needed to extract out the TMCF into 2 nodes.